### PR TITLE
Remove the nova environment file

### PR DESCRIPTION
**Pull request by cloudnull**
This file was only used to get a round a bug. That bug was fixed a long
while ago so this env.d file does nothing.

PR that fixed the bug:
  https://review.openstack.org/#/c/411348/

Signed-off-by: Kevin Carter <kevin.carter@rackspace.com>

Issue: [UG-651](https://rpc-openstack.atlassian.net/browse/UG-651)

### CODE REVIEW

**Comment by cloudnull** [comment]
recheck_cit_all

**Comment by major** [comment]
Just proposed the backport upstream. https://review.openstack.org/#/c/459443/

**Comment by prometheanfire** [comment]
thanks :D

**Comment by major** [comment]
Just bumped the upstream patch to include a link to this PR. This PR should get a nudge once it merges. ;)

**Comment by major** [comment]
@prometheanfire @cloudnull Upstream patch has merged.

**Comment by prometheanfire** [comment]
just need master to consume it then

**Comment by cloudnull** [comment]
recheck_cit_all

**Comment by hughsaunders** [comment]
looks like this needs to be removed from https://github.com/rcbops/rpc-openstack/blob/master/scripts/bootstrap-aio.yml#L126 and https://github.com/rcbops/rpc-openstack/blame/master/rpcd/etc/openstack_deploy/README.rst#L68-L76

**Comment by odyssey4me** [comment]
It looks like this may need a rebase.

File: rpcd/etc/openstack_deploy/env.d/nova.yml [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-container_skel:
-  nova_api_metadata_container:
-    properties:
-      service_name: nova
-      lxc_container_variant: "nova-{{ rpc_release }}"
```
Comment (odyssey4me) [inline rpcd/etc/openstack_deploy/env.d/nova.yml]: The container variant bits are still required until https://github.com/rcbops/rpc-openstack/pull/2375 is finalised and merged, so this content must stay.

The only thing that should be removed is the group membership changes below.

File: scripts/bootstrap-aio.yml [hunk 2]
```diff
@@ -147,7 +147,6 @@
         - name: "env.d/magnum.yml"
         - name: "env.d/memcache.yml"
         - name: "env.d/neutron.yml"
-        - name: "env.d/nova.yml"
```
Comment (odyssey4me) [inline scripts/bootstrap-aio.yml]: This will need to stay for now until https://github.com/rcbops/rpc-openstack/pull/2375 is finalised and merged.

**Comment by alextricity25** [comment]
This has been moved to https://github.com/rcbops/rpc-openstack/pull/2375

**Pull request closed without merging**